components/extractors/communication/ms-outlook/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Microsoft Outlook (Office 365)
+permalink: /components/extractors/communication/ms-outlook/
+redirect_from:
+    - /extractors/communication/ms-outlook/
+---
+
+* TOC
+{:toc}
+
+Microsoft Outlook extractor for Office 365
```

Comment:
@SgtMarmite nejsou tam nejake prerequisites? Je neco potreba povolit na strane Outlook? (e.g. IMAP) Stalo by za to zminit a v uvodu rict, ze je to zalozeny na IMAP